Solutions/Marain.Tenancy.OpenApi.Service/Marain/Tenancy/OpenApi/TenancyServices.yaml

```diff
@@ -280,6 +284,41 @@ components:
               - type: object
               - type: string
       additionalProperties: true
+    UpdateTenantJsonPatchArray:
+      description: An update to a tenant in the form of a JSON Patch document as described by RFC6902
+      type: array
+      items:
+        $ref: "#/components/schemas/UpdateTenantJsonPatchEntry"
+    UpdateTenantJsonPatchEntry:
+      type: object
+      description: An operation describing one change to a tenant as part of an UpdateTenantJsonPatchArray
+      required:
+        - "op"
+        - "path"
+      # Note: http://json.schemastore.org/json-patch uses a mixture of allOf and oneOf to constrain
+      # documents to valid combinations of op and value. (I.e., value should only be present for
+      # add, replace, and test, operations.) However, AutoRest appears not to understand this,
+      # so we're taking a more basic approach here.
+      properties:
+        path: 
+          type: string 
+          description: A JSON-Pointer. Either /name or /properties/propertyName
+        op: 
+          type: string 
+          description: The operation to be performed.
+          # Note: JSON Patch also supports move, copy, and test, but we only support these:
+          enum: [add, replace, remove]
+        value: 
+          description: The value to add or set
+          # We need to set type to none to work around https://github.com/menes-dotnet/Menes/issues/37
+          type: none
```

Comment:
Could you not just omit `type`?

Reply:
No, that was the first thing I tried. And as it says in that bug, Menes has this code:

```csharp
case null: // Lazy people don't say that they are an object
    return JsonObjectType.Object;
```

This is what kicks in if you don't set a `type`. That's why we need to set it explicitly to `none`.